snooty.toml

```diff
@@ -33,3 +34,4 @@ zstdVersion = "com.github.luben:zstd-jni:1.5.5-2"
 logbackVersion = "1.2.11"
 log4j2Version = "2.17.1"
 serializationVersion = "1.5.1"
+bsonVersion = "4.10.0"
```

Comment:
Question: Should the BSON library match the new version 4.10.1 or is it not a matching version?